src/app/auth/(authWithLayout)/sign-in/page.tsx

```diff
@@ -51,8 +52,8 @@ const SignInPage = () => {
           </div>
           <SquareButtonL
             type='submit'
-            backgroundColor={isFormDataValid ? 'bg-main' : 'bg-gray-800'}
             disabled={!isFormDataValid}
+            variant={'primary'}
```

Comment:
```suggestion
            variant={isFormDataValid ? 'primary' : 'tertiaty'}
```

이게 맞지 않나요??
그리고 단일로 사용할 때는 JSX 문법이 아니니 `variant='primary'` 이렇게 사용하시면 됩니다!

원래 틈날 때 디자인 시스템 사용법 간단하게라도 문서화하려고 했는데 수정사항이 워낙 많아서 못했습니다 ㅠㅠ 1차 데모 끝나고라도 틈날 때 해서 공유드릴게요

Reply:
확인했습니다! 이부분만 수정하겠습니다-!